android/UniversalAnalyticsPlugin.java

```diff
@@ -143,13 +144,13 @@ private void trackView(String screenname, CallbackContext callbackContext) {
             return;
         }
 
-        Tracker tracker = GoogleAnalytics.getInstance(this.cordova.getActivity()).getDefaultTracker();
+        Tracker tracker = GoogleAnalytics.getInstance(this.cordova.getActivity()).newTracker(this.trackerId);
```

Comment:
Instead of creating a new tracker every time, maybe use a strategy described here https://developers.google.com/analytics/devguides/collection/android/v4/#tracking-methods